weave/Scripts/Utils/ScoringConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace weave.Utils;
+
+public static class ScoringConstants
+{
+    public const int PointsForSeconds = 25;
+    public const int PointsForLevelUp = 500;
```

Comment:
We should probably have these as floats as we will lose whole digits with delta multiplication

Reply:
Isn't delta double? After the multiplication with delta, the result is stored in _score also as double, so I don't understand why any accuracy would be lost.

Reply:
You're right! My bad :+1: